packages/client/components/AvatarList.tsx

```diff
@@ -29,10 +29,10 @@ const widthToOverlap = {
 // hard coding for now until we have
 // a better solution for handling gutters
 // for a component that may not be there
-const sizeToHeightBump = {
+export const sizeToHeightBump = Object.freeze({
```

Comment:
explicitly use this hack in other place (to be sure don't forget to change in case of refactoring)

Reply:
Why use `Object.freeze` here? What problem does this solve?

Reply:
I used `Object.freeze` as an explicit intent that this object should not be modified in any module that imports it. 
But you're right, we can use a compile-time restriction instead of a run-time limit.